web-platform/glean/0001-Generate-Glean-artifacts-when-building-Rally.patch

```diff
@@ -0,0 +1,85 @@
+From 0965aba3ea952913f5cf6cdde8e5102ac72da548 Mon Sep 17 00:00:00 2001
```

Comment:
Did you mean to include these .patch files?

Reply:
haha no not at all, thank you for catching this